pybmix/core/chain.py

```diff
@@ -0,0 +1,122 @@
+import logging
+import numpy as np
+from google.protobuf.pyext._message import RepeatedScalarContainer
+
+from pybmix.utils.proto_utils import get_field
+import pybmix.proto.matrix_pb2 as matrix_pb2
+
+
+class MmcmChain(object):
```

Comment:
Is it spelled "Mmcm" on purpose...? After the memoizer thingy I can't claim to know anything.
Also, if it is indeed misspelled, I think it looks nicer in all caps